### PR TITLE
finished 3-blocks slider

### PR DESCRIPTION
**Pull request by nmelnychenko**

### CODE REVIEW

File: src/scss/components/Fifth-block/desktop.scss [hunk 1]
```diff
@@ -846,6 +898,102 @@
...
+                        }
+                        .cls-3 {
+                            stroke: #f0578d;
+                        }
+                        .cls-25,
+                        .cls-4 {
+                            opacity: 0.75;
+                        }
+                        .cls-4 {
+                            fill: url(#Безымянный_градиент_5);
```
Comment (StadniukYevhen) [inline src/scss/components/Fifth-block/desktop.scss]: give name to gradient

File: src/scss/components/Fifth-block/desktop.scss [hunk 2]
```diff
@@ -846,6 +898,102 @@
...
+                            stroke: #dd1c51;
+                        }
+                        .cls-19 {
+                            font-size: 18px;
+                        }
+                        .cls-24 {
+                            fill: #79797b;
+                        }
+                        .cls-25 {
+                            fill: url(#Безымянный_градиент_13);
```
Comment (StadniukYevhen) [inline src/scss/components/Fifth-block/desktop.scss]: give name to gradient

File: src/scss/components/Fifth-block/desktop.scss (hunk 1 above)
Comment (nmelnychenko) [reply src/scss/components/Fifth-block/desktop.scss]: done

File: src/scss/components/Fifth-block/desktop.scss (hunk 2 above)
Comment (nmelnychenko) [reply src/scss/components/Fifth-block/desktop.scss]: done

**Pull request merged**